cmake/assets/run_nodered_template.sh.in

```diff
@@ -1,3 +1,6 @@
-cd @EVEREST_DOCKER_DIR@
-docker build -t everest-nodered .
-docker run --rm --network host --name everest_nodered --mount type=bind,source=@FLOW_FILE@,target=/data/flows.json everest-nodered
+docker volume create nodered-config-volume
+docker create -v nodered-config-volume:/data --name nodered-config-container tianon/true:latest true
+docker cp @FLOW_FILE@ nodered-config-container:/data/flows.json
+docker rm nodered-config-container
```

Comment:
would be good to either create named shell function which explains by its name (like copy_file_into_docker_volume) and call it or add some comments above, because it is not that obvious what is happening here.
Wouldn't it be possible to just mount the flows.json directly into the everest_nodered container?  In case it gets modified and we don't want to have it changed in the git, it could also be copied into the build folder beforehand